card/mod/05_standard/set/self/recent.rb

```diff
@@ -1,4 +1,4 @@
-ACTS_PER_PAGE = 10
+ACTS_PER_PAGE = 50
```

Comment:
we actually don't have paging at present, but most of the time in the page is taken up with the initial "all_viewable" query.  (really it's because of the order by)